<launch>
  <node name="camera" pkg="camera" type="camera.py" required="true" output="screen"> 
  <param name="device_num" type = "int" value="0" />
  <param name="width" type = "int" value="1920" />
  <param name="height" type = "int" value="1080" />
  <param name="fps" type = "int" value="30" />
  <rosparam command="load" file="$(find camera)/config/cam.yaml" />
  </node>
</launch>
